pkgs/applications/virtualization/lima/default.nix

```diff
@@ -7,29 +7,34 @@
 , xcbuild
 , sigtool
 , makeWrapper
+, nix-update-script
+, patchelf
+, apple-sdk_13
 }:
 
 buildGoModule rec {
   pname = "lima";
-  version = "0.22.0";
+  version = "1.0.1";
 
   src = fetchFromGitHub {
     owner = "lima-vm";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-ZX2FSZz9q56zWPSHPvXUOf2lzBupjgdTXgWpH1SBJY8=";
+    sha256 = "sha256-XYB8Nxbs76xmiiZ7IYfgn+UgUr6CLOalQrl6Ibo+DRc=";
   };
 
-  vendorHash = "sha256-P0Qnfu/cqLveAwz9jf/wTXxkoh0jvazlE5C/PcUrWsA=";
+  passthru.updateScript = nix-update-script { };
 
-  nativeBuildInputs = [ makeWrapper installShellFiles ]
+  vendorHash = "sha256-nNSBwvhKSWs6to37+RLziYQqVOYfvjYib3fRRALACho=";
+
+  nativeBuildInputs = [ makeWrapper installShellFiles patchelf ]
     ++ lib.optionals stdenv.hostPlatform.isDarwin [ xcbuild.xcrun sigtool ];
 
-  # clean fails with read only vendor dir
+  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [ apple-sdk_13 ];
+
   postPatch = ''
     substituteInPlace Makefile \
-      --replace 'binaries: clean' 'binaries:' \
-      --replace 'codesign --entitlements vz.entitlements -s -' 'codesign --force --entitlements vz.entitlements -s -'
+      --replace-fail 'codesign -f -v --entitlements vz.entitlements -s -' 'codesign -f --entitlements vz.entitlements -s -'
```

Comment:
Does this work properly on `aarch64-darwin`? I thought you could only do this ad‐hoc entitlements stuff on the same machine the resulting binary runs on, or something. (Could be totally wrong.)

Reply:
I am not sure what "works" means here. @emilazy could you please elaborate? I can use the tool to start VMs, but build the derivation on this same machine where I run it, so perhaps I miss something. How do I check if it works? (Note that this `--replace` was present before the bump.)

Reply:
It’s probably fine :) I think I’m probably just thinking of the more exclusive entitlements.